Bottom: 2c2cf28edb548c8ad3097e42aa56e35dd5c6f5ce
Top:    8542cb5b14d2baf7893073dc44c17e580475459c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-03 23:46:21 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
